src/scenes/SelectNextGame/SelectNextGame.css

```diff
@@ -33,8 +33,57 @@
 
 .NextGameName {
   opacity: 0;
-  height: 30px;
+  /* height: 30px; */
```

Comment:
ficou comentado para testes e depois vai voltar a ter o height ou era pra ser removido de vez? se for o segundo caso, eu removeria a linha.

Reply:
Feito